be2-scala/src/test/scala/util/examples/JsonRpcRequestExample.scala

```diff
@@ -10,6 +11,8 @@ import util.examples.Election.OpenElectionExamples._
 import util.examples.Election.SetupElectionExamples.{ELECTION_ID, _}
 import util.examples.Election.EndElectionExamples._
 import util.examples.Election.KeyElectionExamples._
+import util.examples.Election.ResultElectionExamples._
+import util.examples.Election.{ResultElectionExamples, SetupElectionExamples}
```

Comment:
You could remove this line, since it is a repetition of previous lines (line 14 and line 11).

Reply:
Okay, but now why do you include `SetupElectionExamples` ? You already import it in line 10.